modules/ROOT/pages/writing.adoc

```diff
@@ -672,7 +678,253 @@ CREATE CONSTRAINT FOR (p:Person) REQUIRE (p.surname) IS UNIQUE
 
 *Take into consideration that the first label is used for the index creation.*
 
-=== Script option
+
+==== Approach
+
+The connector allows to enforce the following constraints:
+
+.Allowed configuration
+|===
+|config |values |default |description
+
+|`schema.optimization.node` |UNIQUE/KEY/NONE |NONE | Create the `UNIQUE` or `NODE KEY` constraint for the properties defined in the option `node.keys`
+|`schema.optimization.relationship` |UNIQUE/KEY/NONE |false | Create the `UNIQUE` or `NODE KEY` constraint for the properties defined in the option `relationship.keys`
+|`schema.optimization` |TYPE/EXISTS/NONE |NONE | A comma separated list of values. Creates the type constraints for nodes/relationships enforcing the type and non-nullability from the DataFrame schema
+|===
+
+[NOTE]
+Please consider that if you define more than one label we use the *first* for creating constraints.
+
+===== Node constraints
+
+====== Using `schema.optimization.node`
```

Comment:
```suggestion
```

Reply:
Sorry why this?

Reply:
For two reasons:

- It was too deeply nested.
- It was the only section header at that level.